xarray/core/alignment.py

```diff
@@ -169,8 +182,8 @@ def deep_align(objects, join='inner', copy=True, indexes=None,
 
     This function is not public API.
     """
-    from .dataarray import DataArray
-    from .dataset import Dataset
+    from .dataarray import DataArray  # noqa: F811
+    from .dataset import Dataset  # noqa: F811
```

Comment:
This is a workaround to a flake8 limitation https://gitlab.com/pycqa/flake8/issues/553